src/mono/System.Private.CoreLib/src/System/Reflection/Emit/ILGenerator.Mono.cs

```diff
@@ -1026,70 +982,55 @@ internal struct SequencePoint
         public int EndCol;
     }
 
-    internal sealed class Stack
+    internal sealed class Int32Stack
```

Comment:
Could this just use `List<int>` (or derive from `List<int>` if the desire is to add Peek/Pop/Push helpers)?

Reply:
Done.